src/components/Peers.js

```diff
@@ -1,22 +1,22 @@
 import React from "react";
+import { Loading } from "./Loading";
 
-const Peers = ({ stock }) => {
-  return (
-    <div className="peers">
-      <h1 className="title">Top Peers</h1>
-      {stock === null ? (
-        <div className="loading-spinner"></div>
-      ) : stock.length === 0 ? (
-        <div> N/A </div>
-      ) : (
-        <ul className="peers__list">
-          {stock.map(peers => (
+const Peers = ({ Peers }) => {
+  const renderPeersComponent = () => (
+    <ul className="peers__list">
+      {Peers.length !== 0
+        ? Peers.map(peers => (
```

Comment:
`tslint` will tell you off for using the name `Peers` twice for a variable - Once as a prop, and once as the `const Peers` name. It's also harder to understand which you're referring to if this file gets larger.